global.json

```diff
@@ -0,0 +1,5 @@
+{ "sdk": { 
```

Comment:
nit: format

Reply:
Looks like it's regenerated every time a build is triggered so I'll just leave it as is.